repository-data/webfiles/src/main/resources/site/freemarker/intranet/task.ftl

```diff
@@ -131,71 +143,76 @@
+                        <#assign links += [{ "url": "#" + slugify(datesSectionHeader), "title": datesSectionHeader }] />
+
+                        <@stickyNavSections getStickySectionNavLinks({"document": document, "appendSections": links, "includeTopLink": true})></@stickyNavSections>
+
+                        <#-- Restore the bundle -->
+                        <@hst.setBundle basename="intranet.headers, intranet.labels, intranet.task" />
 
-                    <#-- Restore the bundle -->
-                    <@hst.setBundle basename="intranet.headers, intranet.labels, intranet.task" />
                 </div>
-            </div>
 
 
-            <div class="column column--two-thirds page-block page-block--main">
+                <div class="nhsd-t-col-xs-12 nhsd-t-col-s-8">
 
-                <#if document.sections?has_content>
-                    <div class="article-section">
-                        <@sections sections=document.sections />
-                    </div>
-                </#if>
+                    <#if document.sections?has_content>
+                            <@sections sections=document.sections />
+                    </#if>
 
-                <#-- Restore the bundle -->
-                <@hst.setBundle basename="intranet.headers, intranet.labels, intranet.task" />
+                    <#-- Restore the bundle -->
+                    <@hst.setBundle basename="intranet.headers, intranet.labels, intranet.task" />
 
-                <#if hasResponsibleTeams>
-                    <div id="${slugify(responsibleTeamsSectionHeader)}" class="article-section no-border">
-                        <h2>${responsibleTeamsSectionHeader}</h2>
-                        <ul>
-                            <#list document.responsibleTeams as team>
-                                <li><a href="<@hst.link hippobean=team />">${team.title}</a></li>
-                            </#list>
-                        </ul>
-                    </div>
-                </#if>
-
-                <div class="article-section" id="${slugify(datesSectionHeader)}">
-                    <h2>${datesSectionHeader}</h2>
-                    <div class="detail-list-grid detail-list-grid--regular">
-                        <#if document.reviewDate?has_content>
-                        <dl class="detail-list">
-                            <dt class="detail-list__key"><@fmt.message key="labels.review-date"/></dt>
-                            <dd class="detail-list__value"><@fmt.formatDate value=document.reviewDate.time?date type="date" pattern="d MMMM yyyy" timeZone="${getTimeZone()}" /></dd>
-                        </dl>
-                        </#if>
+                    <#if hasResponsibleTeams>
+                        <div id="${slugify(responsibleTeamsSectionHeader)}"
+                             class="article-section no-border">
+                            <h2>${responsibleTeamsSectionHeader}</h2>
+                            <ul>
```

Comment:
If this is bullet list needs class, "nhsd-t-list nhsd-t-list--bullet"

Reply:
done